_data/contributors.yml

```diff
@@ -279,7 +279,25 @@
       mentors: Vassil Vassilev, David Lange
       report: |
         [URL](https://indico.cern.ch/event/1066812/contributions/4495290/attachments/2302817/3917355/Iris%20HeP.pdf"){:target="_blank"}
-
+        
+- name: Surya Somayyajula
```

Comment:
Can you move this entry just after the entry for Ioana and make sure the description is aligned at 80 columns. Please squash all commits into one.

Reply:
Just did it